### PR TITLE
solves ansible/ansible#79702 and #1518 

### PR DESCRIPTION
**Pull request by marek22k**
…thub.com/netblue30/firejail/issues/1518 for ansible

Signed-off-by: Marek Küthe <m.k@mk16.de>

### CODE REVIEW

File: etc/inc/allow-ssh.inc [hunk 1]
```diff
@@ -2,6 +2,8 @@
 # Persistent customizations should go in a .local file.
 include allow-ssh.local
 
+# Ansible
+noblacklist ${HOME}/.ansible/cp
```
Comment (rusty-snake) [inline etc/inc/allow-ssh.inc:6]: Never blacklisted.

**Review by rusty-snake — CHANGES_REQUESTED** [review]

File: etc/inc/allow-ssh.inc (hunk 1 above)
Comment (marek22k) [reply etc/inc/allow-ssh.inc:6]: Could you expand on that a little bit? For example, why is `.ssh` inside, the same logic should apply to `.ssh`. Please note that this file is included by another one, which means that not all blacklists are in this file.

Comment (rusty-snake) [reply etc/inc/allow-ssh.inc:6]: No file contains a blacklist for `.ansible/cp`, therefore it does not make sense to nobalcklist it. It is a no-op.

`.ssh` is blacklisted (in contrast to `.ansible/cp`), see
https://github.com/netblue30/firejail/blob/a4eb84d5e8165586a59d5e8e30c771fb7f133e2f/etc/inc/disable-common.inc#L444

Having the blacklist and noblacklist in the same file would be nonsense anyway and should never be the case.

Where is the blacklist

**Review by rusty-snake — COMMENTED** [review]
&nbsp;

File: etc/inc/allow-ssh.inc (hunk 1 above)
Comment (rusty-snake) [reply etc/inc/allow-ssh.inc:6]: Maybe check `grep ansible ~/.config/firejail/*`.

Comment (marek22k) [reply etc/inc/allow-ssh.inc:6]: Ahh, I see. But somehow accessing this file with firejail seems to be problematic. Ansible with SSH does not work at least.

Comment (rusty-snake) [reply etc/inc/allow-ssh.inc:6]: You can use `--debug` to geht more information. Maybe a symlink? And use firemon to see which sandboxes are started.

**Pull request closed without merging**